src/sentry/replays/usecases/summarize.py

```diff
@@ -247,42 +247,53 @@ def get_summary_logs(
     error_events: list[EventDict],
     project_id: int,
     is_mobile_replay: bool = False,
+    replay_start: str | None = None,
 ) -> list[str]:
     # Sort error events by timestamp. This list includes all feedback events still.
     error_events.sort(key=lambda x: x["timestamp"])
-    return list(generate_summary_logs(segment_data, error_events, project_id, is_mobile_replay))
+    return list(
+        generate_summary_logs(
+            segment_data, error_events, project_id, is_mobile_replay, replay_start
```

Comment:
```suggestion
            segment_data, error_events, project_id, is_mobile_replay=is_mobile_replay, replay_start=replay_start
```
and same for L598 - best to be explicit w kwargs